### DISCUSSION TITLE
My Little-Little Layout Field

### DISCUSSION THREAD

**Discussion by marknotton** [wbrowar/craft-little-layout#4] [Ideas]
I really like this plugin! It's intuitive and easy to get on with. I wanted to share my use-case and how I've made an already minimalist field type even more minimalist.

With a little bit of CSS I already had rendering via my own private modules AssetBundler -- I positioned the Little Layout field next to the "action buttons" on my [SuperTable](https://verbb.io/craft-plugins/super-table/features) field.   

![SuperTable](https://i.ibb.co/fMhTcrj/Screenshot-2021-04-07-at-7-43-22-am.png)

By relocating the Little Layout field to the same place for all my block types we found the client was better able to see how content might be rendered at a glance. There's nothing wrong with how it works now, it's just I wanted to keep each block as clean as possible.

**Comment by wbrowar** [comment]
This is _very_ cool. Thanks for sharing, @marknotton!
